config/routes.rb

```diff
@@ -2,6 +2,9 @@
 
 Rails.application.routes.draw do
   devise_for :users
+  scope '/students/:id' do
+    resources :focus, only: [:index]
+  end
   resources :students, only: %i[index show]
```

Comment:
Instead of a separate scope, you can put it all under students
```suggestion
  resources :students, only: %i[index show] do
    resources :focuses
  end
```

Reply:
Also make sure that the name of the resource is plural! `focuses`